### ISSUE TITLE
Provide full source code in releases

### ISSUE THREAD

**Issue by amirmahdi-nsh** [23rd/tdesktop#43]
Hello, I was trying to build Forkgram from source in Arch Linux.
As probably you know download classic source code section in releases does not contain Git Submodules. So build from source requires clone repo with git and nor `zip` or `tar.gz` source code، for this reason original Telegram Desktop always put full source beside the executable files.

I ask you to do the same :)
Thank you. Sorry if I have some grammatical mistake.

**Comment by 23rd** [comment]
Hello, are you sure you need the full source code?
If you want just use my fork in Linux, you can download [this](https://github.com/Forkgram/tdesktop/releases/download/v1.9.17/Telegram.tar.xz) version and log in. Then the application will be updated to the latest version.

**Comment by amirmahdi-nsh** [comment]
> Hello, are you sure you need the full source code?

Hello, Yes I'm sure. It can also be done with git clone the repo but it's a bit not standard and hard to implement in AUR. in addition git bandwidth consumption is much more than download only full source code file

> If you want just use my fork in Linux, you can download this version and log in

Thanks but I want to build from source ;)

**Comment by 23rd** [comment]
Please check the latest release.

**Comment by amirmahdi-nsh** [comment]
Thank you for your help and attention.
Unfortunately, it give error during the compile :(
[here](https://pastebin.com/raw/kH5gj8uA) is the log

**Comment by 23rd** [comment]
First of all, did you test the full code from the official repository?

**Comment by 23rd** [comment]
I have checked builds with the full sources in Windows and Linux, and they was successful.

**Comment by amirmahdi-nsh** [comment]
> First of all, did you test the full code from the official repository?

Due to my hard disk space it stops to compiling when almost half files are compiled successfully. But Forkgram errors just some minutes after starting.

> I have checked builds with the full sources in Windows and Linux, and they was successful.

Thanks. So, I will test it on another distro tomorrow

**Comment by amirmahdi-nsh** [comment]
Yes, I tried it and you are right. I found where the problem was.
I was trying to compile the stable official telegram desktop and it compiles successfully but now I realized I can't compile offical telegram desktop pre-release 2.5.6 and I think that why I'm usage the stable tg_owt library and for compile newer telegram desktop I have to install the newer tg_owt.
So, probably this problem should solve if I wait 2.5.6 be stable or install unstable tg_owt and then again try to compile.

Thank you very much for supporting and answering me :)
